feature_engine/selection/drop_features.py

```diff
@@ -5,6 +5,9 @@
 from feature_engine.dataframe_checks import _is_dataframe
 from feature_engine.selection.base_selector import BaseSelector
 from feature_engine.validation import _return_tags
+from feature_engine.variable_manipulation import (
```

Comment:
I see the point and I was in doubt about adding the functionality here. The reasoning I made was the following.

- During development, a user realises that all variables related to socio_demographic are unreliable. (S)he then define a list of variables to drop like var_to_drop = [col for col in df.columns if col.startswith(socio_demographic)].

- One of this variable is constant. For example a variable saying if the customer is above 18 years (image all customers is scope of the model are).

- A pipeline DropFeatures(var_to_drop) + DropConstantFeatures would work but a pipeline DropConstantFeatures + DropFeatures(var_to_drop) would fail.

Personally, I would always put the DropFeatures as the first part of a pipeline so I am completely fine with the revert you did.